### ISSUE TITLE
[Bug]: Extension doesn't work again

### ISSUE THREAD

**Issue by Pdkl** [yakisova41/return-youtube-comment-username#105]
### Browser

Vivaldi 5.6.2867.62

### Release Type

Chrome Extension

### The URL where the problem occurred

https://www.youtube.com/watch?v=b_YIofkDl7o

### Screenshot

Debugging information is absent.
![image](https://github.com/yakisova41/return-youtube-comment-username/assets/163922991/09e8bba0-5cec-4bbc-b074-0d141a2fcb06)

### What is the problem?

YouTube broke it all again. Usernames are not available. Extension is not working at all.
![image](https://github.com/yakisova41/return-youtube-comment-username/assets/163922991/baba206b-5c2e-474e-8915-247cc6b63265)

**Comment by yakisova41** [comment]
Does debug info really not exist?
If you cant found info, probably the extension doesnt loaded.
Please check.

![スクリーンショット 2024-04-13 074132](https://github.com/yakisova41/return-youtube-comment-username/assets/75610521/d36a3fdb-1526-4d2b-8662-04282ab83b4c)

**Comment by Pdkl** [comment]
> Does debug info really not exist? If you cant found info, probably the extension doesnt loaded or it is old version. Please check.

I found debug info in Extensions page when activated Developer's mode and pressed the Errors button.
![image](https://github.com/yakisova41/return-youtube-comment-username/assets/163922991/d64ab0d9-e6bc-402f-ac3b-f7528d3ca9f5)
![image](https://github.com/yakisova41/return-youtube-comment-username/assets/163922991/d73cc318-07ed-4729-aaee-b60c59ca7062)
In Developer's tools console, it looks like the same error.
![image](https://github.com/yakisova41/return-youtube-comment-username/assets/163922991/1bbda486-cbad-4f36-a233-70fa561d2cb3)
```
    debugLog(
      `Return Youtube comment Username v${package_default.version}`,
      [
        "",
        `PAGE_BUILD_LABEL: ${window.yt.config_.PAGE_BUILD_LABEL}`,
        `INNERTUBE_CLIENT_VERSION: ${window.yt.config_.INNERTUBE_CLIENT_VERSION}`,
        `INNERTUBE_CONTEXT_CLIENT_VERSION: ${window.yt.config_.INNERTUBE_CONTEXT_CLIENT_VERSION}`,
        `INNERTUBE_CONTEXT_GL: ${window.yt.config_.INNERTUBE_CONTEXT_GL}`,
        `Browser: ${window.yt.config_.INNERTUBE_CONTEXT.client.browserName} v${window.yt.config_.INNERTUBE_CONTEXT.client.browserVersion}`,
        `Login: ${window.yt.config_.LOGGED_IN}`,
        `Href: ${location.href}`
      ].join("\n")
    );
```
 Deleting and reinstalling the extension didn't help.

**Comment by yakisova41** [comment]
probably fixed it in v0.5.1.
Please wait to release in chrome webstore
